bgbackup.sh

```diff
@@ -502,7 +504,7 @@ fi
 mysqlcreate
 
 # Check that mysql client can connect
-$mysqlcommand "SELECT 1 FROM DUAL"
+$mysqlcommand "SELECT 1 FROM DUAL" 1>/dev/null
```

Comment:
Looking at the code it just looks at the exit code. I do not think such test is needed for fixing this bug.